.github/workflows/tests_branch.yml

```diff
@@ -26,11 +21,30 @@ jobs:
       - name: black linter
         run: black --diff --check ./
 
-  some_unit_tests:
+  branch_unit_tests:
     name: Unit tests
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v3
-      - name: some tests
+      - name: some tests [PLACEHOLDER]
         run: |
-          echo "running tests"
+          echo "running tests ..."
+          # TODO add pytest tests
+
+  branch_devcontainer_tests:
+    name: Devcontainer tests
+    runs-on: ubuntu-20.04
```

Comment:
Upgrade to 22.04 or write a note saying that this is available and fully supported via GitHub Actions?